ambient-poly.py

```diff
@@ -6,8 +6,7 @@
 import polyinterface
 import sys
 import time
-#import httplib
-import urllib3
+import requests
```

Comment:
Changing to use requests seems reasonable, but I'd prefer that this change be a separate commit from one that adds support for the additional sensors.